docs/source/getting_started/cmakepp_examples/classes.rst

```diff
@@ -145,7 +145,7 @@ prepended to their name. Here is the function:
       Automobile(GET "${self}" _ds color km_driven)
 
       # Print out a message
-      message("I am an automobile, I am ${_ds_color}, and I have driven ${km_driven} km.")
+      message("I am an automobile, I am ${_ds_color}, and I have driven ${_ds_km_driven} km.")
```

Comment:
If it's all the same to you, when you find a code snippet in the docs. Move it to `tests/docs/` and then include the relevant snippet in the documentation via the `literalinclude` directive (see [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude)) so that the examples are actually unit tested.

Reply:
Sure. I was actually planning to ask you about where to put the code snippets! I will do that as well in this PR.

Reply:
I updated the initial comment with this task and am going to resolve this conversation now.